### ISSUE TITLE
Incompatabilities between lit0.13 and lit0.14

### ISSUE THREAD

**Issue by wgottschalk** [matthewp/haunted#38]
I tried running one of the demos and it took me a second to understand what was going wrong. I used the following code:
```js
<!doctype html>
<html lang="en">
<my-counter></my-counter> 
<script type="module">
  import { html as lithtml } from 'https://unpkg.com/lit-html/lit-html.js';
  import { component, useState, html as hauntedhtml} from 'https://unpkg.com/haunted/haunted.js'; 

  function Counter() {
    const [count, setCount] = useState(0); 
    return lithtml`
      <div id="count">${count}</div>
      <button type="button" @click=${() => setCount(count + 1)}>Increment</button>
    `;
  } 

  customElements.define('my-counter', component(Counter));
</script>
```
and got this as my render output:
![image](https://user-images.githubusercontent.com/11728959/49424753-647a9680-f750-11e8-8e30-e025455ca142.png)

Switching to `hauntedhtml` works fine. From what I can tell, it looks like lit is on v0.14 while haunted is using 0.13. Not sure what changed between the two version but I just wanted to give a heads up! My time is a little limited right now but I can try and pitch in if the change is simple.

**Comment by wgottschalk** [comment]
Also note lit's release page https://github.com/Polymer/lit-html/releases

**Comment by matthewp** [comment]
Thanks for the report. I'll upgrade to lit 0.14 now. It will be a breaking change fwiw.

**Comment by matthewp** [comment]
This was released as Haunted 3.0
